src/openforms/api/serializers.py

```diff
@@ -69,6 +75,27 @@ class ValidationErrorSerializer(ExceptionSerializer):
     invalid_params = FieldValidationErrorSerializer(many=True)
 
 
+@dataclass
+@total_ordering
+class BatchItem:
```

Comment:
Maybe `_BatchItem` is better, this should not be used elsewhere, and should be kept as an implementation detail.